x-pack/test_serverless/api_integration/test_suites/observability/config.feature_flags.ts

```diff
@@ -22,13 +22,12 @@ export default createTestConfig({
   // add feature flags
   kbnServerArgs: [
     '--xpack.infra.enabled=true',
-    '--xpack.security.roleManagementEnabled=true', // enables custom roles
-    `--xpack.spaces.maxSpaces=100`, // enables spaces UI capabilities
+    '--xpack.security.roleManagementEnabled=true', // needed to check composite feautures in /observability/platform_security/authorization.ts
```

Comment:
@azasypkin Should we expose the `api/security/privileges` endpoint in serverless even if role management is off? Currently, we do not.

Reply:
Hmm, I’d say no, we shouldn’t. But would you mind giving me a bit of context? Do I understand correctly that we’ll still keep role/privilege-related tests for O11y, but only via the “feature_flags” config (unlike other project types for which we can finally move to base/normal configs)?

Reply:
> but this endpoint is only available when we enable role management so it needs to remain in the feature flag config until either oblt enabled custom roles

Ah, got it, thanks, it makes sense to me 👍